proto/Networking/Responses/BelugaTransactionStartResponse.proto

```diff
@@ -24,12 +21,4 @@ message BelugaTransactionStartResponse {
 		ERROR_TOO_MANY_POKEMON = 9;
 		ERROR_NO_POKEMON_SPECIFIED = 10;
 	}
-
-	message BelugaBleTransferPrep {
```

Comment:
Why removing it from there? I don't don't think it's used elsewhere

Reply:
dans dossier data/beluga avec le reste qaund j'ai ajouter sa j'ai mis comme sa

Reply:
c'est bon dans data/beluga ???? je ne l'ai pas changer dans le dernier commit